README.md

```diff
@@ -1,25 +1,34 @@
 # Simple Settings
 
-A template to get started with Nextcloud app development.
+Minimal settings for Nextcloud.
 
-## Usage
 
-- To get started easily use the [Appstore App generator](https://apps.nextcloud.com/developer/apps/generate) to
-  dynamically generate an App based on this repository with all the constants prefilled.
-- Alternatively you can use the "Use this template" button on the top of this page to create a new repository based on
-  this repository. Afterwards adjust all the necessary constants like App ID, namespace, descriptions etc.
+## Build
 
-Once your app is ready follow the [instructions](https://nextcloudappstore.readthedocs.io/en/latest/developer.html) to
-upload it to the Appstore.
+```
+$ npm run build
+```
 
-## Resources
 
-### Documentation for developers:
+## Translation workflow
 
-- General documentation and tutorials: https://nextcloud.com/developer
-- Technical documentation: https://docs.nextcloud.com/server/latest/developer_manual
+1. Search for translations and create `.pot` file using `translationtool.phar create-pot-files`
 
-### Help for developers:
+   ```bash
```

Comment:
* I would use `shell` instead of `bash` to keep it more generic

Reply:
This line was removed. It was part of the generated template app.